### PR TITLE
Add token names to registry

### PR DESCRIPTION
**Pull request by graemecode**

### CODE REVIEW

File: contracts/TokenRegistry.sol [hunk 1]
```diff
@@ -142,4 +159,20 @@ contract TokenRegistry is Ownable {
...
+     *   getTokenNameByIndex(1);
+     *   => "Canonical Wrapped Ether"
+     */
+    function getTokenNameByIndex(uint _index) public view returns (string) {
+        string memory symbol = getTokenSymbolByIndex(_index);
+
+        string memory tokenName = getTokenNameBySymbol(symbol);
+
+        return tokenName;
+    }
```
[the remark this answers is not in the document]
Comment (graemecode) [reply contracts/TokenRegistry.sol]: Done!

**Comment by graemecode** [comment]
♻️  @nadavhollander

File: test/ts/integration/token_registry.ts [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+            });
+        });
+    });
+
+    describe("#getTokenAddressBySymbol", () => {
+        describe("when given 'WETH'", () => {
+            it("returns a valid address", async () => {
+                const address = await registry.getTokenAddressBySymbol.callAsync("WETH");
+
+                expect(address.length).to.eq(42);
```
Comment (nadavhollander) [inline test/ts/integration/token_registry.ts]: This should test for something other than length -- this test will pass silently even if `address(0)` is returned, which is what would happen when an untracked symbol is queried.

Comment (nadavhollander) [reply test/ts/integration/token_registry.ts]: e.g. `.not.eq(NULL_ADDRESS)`

File: test/ts/integration/token_registry.ts [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+
+    describe("#getTokenIndexBySymbol", () => {
+        describe("when given 'WETH'", () => {
+            it("returns a number", async () => {
+                const index = await registry.getTokenIndexBySymbol.callAsync("WETH");
+
+                expect(index.toNumber()).to.be.at.least(0);
+            });
+        });
+    });
```
Comment (nadavhollander) [inline test/ts/integration/token_registry.ts]: I believe `getTokenSymbolByIndex` is missing.  Please add.

Comment (nadavhollander) [reply test/ts/integration/token_registry.ts]: Likewise w/ `getNumDecimalsByIndex` and `getTokenAttributesBySymbol|Index`

**Comment by nadavhollander** [comment]
Awesome work :electron: LGTM!

**Pull request merged**